h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -148,32 +151,8 @@ public StackedEnsembleParametersV99 fillFromImpl(StackedEnsembleModel.StackedEns
           }
         }
         
-        ModelParametersSchemaV3 paramsSchema;
-        Model.Parameters params;
-        switch (metalearner_algorithm) {
-          case AUTO:
-          case glm:
-            paramsSchema = new GLMV3.GLMParametersV3();
-            params = new GLMModel.GLMParameters();
-            // FIXME: This is here because there is no Family.AUTO. It enables us to know if the user specified family or not.
-            // FIXME: Family.AUTO will be implemented in https://0xdata.atlassian.net/projects/PUBDEV/issues/PUBDEV-7444
-            ((GLMModel.GLMParameters) params)._family = null;
```

Comment:
This was supposed to be fixed in different PR[1] but I forgot about it and it is handy to have here fixed already so I added the fix to this PR.

[1] https://github.com/h2oai/h2o-3/pull/4760 (now CLOSED PR)